pytorch_toolkit/ote/tools/export.py

```diff
@@ -25,6 +26,9 @@ def main():
     arg_parser = build_arg_parser(modules['arg_parser'])
     ote_args = vars(arg_parser.get_export_parser(MODEL_TEMPLATE_FILENAME).parse_args())
 
+    config_transformers_engine = ConfigTransformersEngine(MODEL_TEMPLATE_FILENAME, modules.get("config_transformers"))
```

Comment:
will it work if the model has not been compressed at all (just trained)?

Reply:
Fixed